.rubocop.yml

```diff
@@ -17,10 +18,18 @@ Metrics/MethodLength:
 Metrics/BlockLength:
   Exclude:
     - 'spec/**/*.rb'
+    - 'api_auth.gemspec'
```

Comment:
FYI: I suppose that big block in `api_auth.gemspec` it's ok, so I disabled this rule for this file.

Reply:
:+1: